### ISSUE TITLE
Pylint fails with "ImportError: cannot import name 'version'"

### ISSUE THREAD

**Issue by musabhusaini** [PyCQA/pylint#4402]
Running pylint 2.3 on any set of files fails. This issue is fixed if astroid 2.5.3 is explicitly installed.

### Steps to reproduce

Run pylint 2.3 on any set of files.

### Current behavior

This error is received:

```
Traceback (most recent call last):
  File "/usr/local/bin/pylint", line 8, in <module>
    sys.exit(run_pylint())
  File "/usr/local/lib/python3.6/site-packages/pylint/__init__.py", line 17, in run_pylint
    from pylint.lint import Run
  File "/usr/local/lib/python3.6/site-packages/pylint/lint.py", line 77, in <module>
    from astroid.__pkginfo__ import version as astroid_version
ImportError: cannot import name 'version'
```

This is part of our CI checks and we just started getting this error a few hours ago.

### Expected behavior

Linting should proceed without runtime errors.

### pylint --version output

Result of `pylint --version` output:

```
Traceback (most recent call last):
  File "/usr/local/bin/pylint", line 8, in <module>
    sys.exit(run_pylint())
  File "/usr/local/lib/python3.6/site-packages/pylint/__init__.py", line 17, in run_pylint
    from pylint.lint import Run
  File "/usr/local/lib/python3.6/site-packages/pylint/lint.py", line 77, in <module>
    from astroid.__pkginfo__ import version as astroid_version
ImportError: cannot import name 'version'
```

**Comment by Pierre-Sassoulas** [comment]
Thank you for the report, it's going to be fixed in astroid 2.5.6.

**Comment by musabhusaini** [comment]
Thanks for the quick turnaround.
